# failable_response

This code doesn't work yet, since it's waiting on PEP 696 (https://github.com/python/steering-council/issues/177).

This is just a reminder to make this into a repo later.
